fastchat/serve/vllm_worker.py

```diff
@@ -13,7 +13,7 @@
 from fastapi.responses import StreamingResponse, JSONResponse
 import torch
 import uvicorn
-from vllm import AsyncLLMEngine
+from vllm.engine.async_llm_engine import AsyncLLMEngine
```

Comment:
use try except to make it work with both the old version and the new version

Reply:
<img width="601" alt="image" src="https://github.com/lm-sys/FastChat/assets/4583537/e1618bd3-bda3-4b74-ba3e-bf07e375d495">
test pass